src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -281,33 +340,28 @@ export class DotEditContentHtmlService {
         dotEditContentletEl.dataset.dotIdentifier = contentlet.identifier;
         dotEditContentletEl.dataset.dotInode = contentlet.inode;
         dotEditContentletEl.dataset.dotType = contentlet.type;
+        dotEditContentletEl.dataset.dotBasetype = contentlet.baseType;
 
         /*
             TODO: we have the method: DotEditContentToolbarHtmlService.addContentletMarkup that does this, we need
             to consolidate this.
         */
         dotEditContentletEl.innerHTML = `<div class="dotedit-contentlet__toolbar">
```

Comment:
Why don't you use this: https://github.com/dotCMS/core-web/pull/531/files#diff-88e1c17d852c4afe2f34435e8ffd784aR90 here?